sqs.tf

```diff
@@ -0,0 +1,29 @@
+resource "aws_sqs_queue" "keycloak_to_alerts_concierge_user_updates" {
+  name = "keycloak-${var.environment}-alerts-concierge-user-updates"
+  sqs_managed_sse_enabled = true
+}
+
+# allow Keycloak to publish messages to SQS
+resource "aws_sqs_queue_policy" "keycloak_to_alerts_concierge_user_updates" {
+  queue_url = aws_sqs_queue.keycloak_to_alerts_concierge_user_updates.id
+  policy    = data.aws_iam_policy_document.keycloak_to_alerts_concierge_user_updates.json
+}
+resource "aws_iam_role_policy" "keycloak_to_alerts_concierge_user_updates_publish" {
```

Comment:
Can you move this policy to [`iam.tf`](https://github.com/mbta/terraform-keycloak-sso/blob/main/iam.tf) so it's alongside the other IAM policies for Keycloak?